### PR TITLE
PHP homework

### PR DESCRIPTION
**Pull request by Misakiiii-333**
php homework

### CODE REVIEW

**Comment by pa-vuhn** [comment]
1. Please check your source code format and fix the format following PSR-2 standard at https://www.php-fig.org/psr/psr-2/ carefully.
2. Move all files to a folder `php-homework`

File: php-homework3.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ }
+ if ($y == 2){
+     return true;
+     //2は素数
+ }
+
+ $x = $y/2 ;
+ for ($i = 2;$i <= $x; $i++){
+     if($y%$i == 0){
+     return false;
```
Comment (pa-vuhn) [inline php-homework3.php:16]: The indentation is wrong, please fix.

File: php-homework3.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     return true;
+     //2は素数
+ }
+
+ $x = $y/2 ;
+ for ($i = 2;$i <= $x; $i++){
+     if($y%$i == 0){
+     return false; 
+     //数値xが2以上のもので2で割り切れるものは素数でない
+ }
```
Comment (pa-vuhn) [inline php-homework3.php:18]: The indentation is wrong, please fix.

File: php-homework3.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ }
+  return true;
```
Comment (pa-vuhn) [inline php-homework3.php:20]: The indentation is wrong, please fix.

File: php-homework3.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+
+if(Primenumber(9)){
+  echo "9 is prime number"."\n";
```
Comment (pa-vuhn) [inline php-homework3.php:25]: The indentation is wrong, please fix.

File: php-homework4.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<?php
+ 
+ function isPrimenumber($y){
```
Comment (pa-vuhn) [inline php-homework4.php:3]: There is a space at beginning of the line, please remove

File: php-homework4.php [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ }
+
+ $x = $y/2 ;
+ for ($i = 2;$i <= $x; $i++) {
+     if($y%$i == 0){
+     return false; //数値xが2以上のもので2で割り切れるものは素数でない
+    }
+ }
+  return true;
+}
```
Comment (pa-vuhn) [inline php-homework4.php:18]: The indentations of above function is almost wrong, please fix all.

File: php-homework1.php [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<?php
+$sum = 0;
+for ($i = 0; $i < 50; $i++) {
+    if ($i%2 === 1) {
+    $sum += $i;
```
Comment (pa-vuhn) [inline php-homework1.php:5]: Add a indent here (4 spaces following PSR-2 standard)

File: php-homework2.php [hunk 8]
```diff
@@ -0,0 +1,15 @@
+<?php 
+
+function factorialOfNumber($n) 
+{
+   $result = 1;
+   
+   for ($i = 2; $i <= $n; $i++) {
+
```
Comment (pa-vuhn) [inline php-homework2.php:8]: remove this blank line

**Comment by Misakiiii-333** [comment]
I corrected mistakes about php-homework  and made a "php-homework"folder .
Ans I've consolidated the files into folders .

File: php-homework/PHP-homework1.php [hunk 9]
```diff
@@ -0,0 +1,10 @@
+<?php
+$sum = 0;
+for ($i = 0; $i < 50; $i++) {
+    if ($i%2 === 1) {
+        $sum += $i;
+   }
```
Comment (pa-vuhn) [inline php-homework/PHP-homework1.php:6]: The indent is not correct, please fix.

File: php-homework/PHP-homework3.php [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+        return true;
+ }
+    $x = $y/2;
+    for ($i = 2;$i <= $x; $i++){
+        if($y%$i == 0){
+            return false; 
+    }
+ }
+    return true;
+}
```
Comment (pa-vuhn) [inline php-homework/PHP-homework3.php:17]: Please fix indentation for the function

File: php-homework/PHP-homework4.php [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+ }
+
+    $x = $y/2 ;
+    for ($i = 2;$i <= $x; $i++) {
+         if($y%$i == 0){
+            return false; 
+    }
+}
+    return true;
+}
```
Comment (pa-vuhn) [inline php-homework/PHP-homework4.php:18]: Please fix indentation for the function

**Comment by Misakiiii-333** [comment]
I'm sorry for repeating myself.　I corrected mistakes about php-homework .

File: php-homework/PHP-homework2.php [hunk 12]
```diff
@@ -0,0 +1,14 @@
+<?php 
+
+function factorialOfNumber($n) 
+{
+    $result = 1;
+    for ($i = 2; $i <= $n; $i++) {
+       
+        $result *= $i ;
+    }
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework2.php:9]: remove blank line

File: php-homework/PHP-homework3.php (hunk 10 above)
Comment (pav-thanhtt) [reply php-homework/PHP-homework3.php:17]: use `===` instead of `==`

File: php-homework/PHP-homework3.php [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+    for ($i = 2;$i <= $x; $i++){
+        if($y%$i == 0){
+            return false; 
+        }
+    }
+    return true;
+}
+
+
+if(Primenumber(9)){
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php:20]: please add space like below
`if (Primenumber(9)) {`

File: php-homework/PHP-homework4.php (hunk 11 above)
Comment (pav-thanhtt) [reply php-homework/PHP-homework4.php:18]: use `===` instead of `==`

File: php-homework/PHP-homework4.php [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+    $x = $y/2 ;
+    for ($i = 2;$i <= $x; $i++) {
+        if($y%$i == 0){
+            return false; 
+        }
+    }
+    return true;
+}
+
+function getPrimeNumbers($ty){
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework4.php:20]: add space like below
`function getPrimeNumbers($ty) {`

File: php-homework/PHP-homework4.php [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+    $result=[] ;
+    for ($i = 1 ; $i<= $ty ; $i++){
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework4.php:22]: update
`for ($i = 1; $i <= $ty; $i++){`

File: php-homework/PHP-homework3.php (hunk 10 above)
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php]: use `===` instead of `==`

File: php-homework/PHP-homework4.php [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+    if ($y == 2){
+        return true; 
+ }
+
+    $x = $y/2 ;
+    for ($i = 2;$i <= $x; $i++) {
+         if($y%$i == 0){
+            return false; 
+    }
+}
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework4.php]: use `===` instead of `==`

File: php-homework/PHP-homework1.php [hunk 17]
```diff
@@ -0,0 +1,10 @@
+<?php
+$sum = 0;
+for ($i = 0; $i < 50; $i++) {
+    if ($i%2 === 1) {
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework1.php:4]: update like below
`if ($i % 2 === 1) {`

File: php-homework/PHP-homework3.php [hunk 18]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+function Primenumber($y) {
+    if($y===0 || $y===1) {
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php:4]: update like below
`if($y === 0 || $y === 1) {`

File: php-homework/PHP-homework3.php [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+        return false;
+ }
+    if ($y === 2) {
+        return true;
+ }
+    $x = $y/2;
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php:10]: update like below
`$x = $y / 2`

File: php-homework/PHP-homework3.php [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+    for ($i = 2;$i <= $x; $i++) {
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php:11]: update like below
`for ($i = 2; $i <= $x; $i++) {`

File: php-homework/PHP-homework3.php [hunk 21]
```diff
@@ -0,0 +1,31 @@
...
+        if($y%$i === 0) {
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php:12]: update like below
`if($y % $i === 0) {`

File: php-homework/PHP-homework4.php [hunk 22]
```diff
@@ -0,0 +1,32 @@
...
+    if ($y === 2) {
+        return true; 
+ }
+
+    $x = $y/2 ;
+    for ($i = 2;$i <= $x; $i++) {
+        if($y%$i === 0) {
+            return false; 
+        }
+    }
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework4.php:16]: update like below
```
    $x = $y / 2 ;
    for ($i = 2; $i <= $x; $i++) {
        if($y % $i === 0) {
            return false; 
        }
    }
```

**Comment by pav-thanhtt** [comment]
Can you check and follow this Coding Style Guide
https://www.php-fig.org/psr/psr-2/

**Comment by Misakiiii-333** [comment]
I am so sorry. I carefully fixed it, comparing it to the coding guide.
Please review PR...

File: php-homework/PHP-homework3.php [hunk 23]
```diff
@@ -0,0 +1,31 @@
...
+function Primenumber($y) {
+    if($y === 0 || $y === 1) {
+        return false;
+ }
+    if ($y === 2) {
+        return true;
+ }
+    $x = $y / 2
+    for ($i = 2; $i <= $x; $i++) {
+        if($y % $i === 0) {
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php:12]: update like below
`if ($y % $i === 0) {`

File: php-homework/PHP-homework4.php [hunk 24]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+function isPrimenumber($y) {
+    if($y === 0 ||$y === 1) {
+        return false; 
+ }
+    if ($y === 2) {
+        return true; 
+ }
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework4.php:9]: same as above

File: php-homework/PHP-homework4.php [hunk 25]
```diff
@@ -0,0 +1,32 @@
...
+
+    $x = $y / 2 ;
+    for ($i = 2; $i <= $x; $i++) {
+        if($y % $i === 0) {
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework4.php:13]: same as above

File: php-homework/PHP-homework3.php [hunk 26]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+function Primenumber($y) {
+    if($y === 0 || $y === 1) {
+        return false;
+ }
+    if ($y === 2) {
+        return true;
+ }
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php:9]: update like below
```
    if ($y === 0 || $y === 1) {
        return false;
    }
    if ($y === 2) {
        return true;
    }
```

Comment (Misakiiii-333) [reply php-homework/PHP-homework3.php]: It seems that when I copy-pasted something I wrote elsewhere(VS code Editor), it wasn't officially displayed（blank space）.
please review PR... I'm so sorry...

File: php-homework/PHP-homework3.php [hunk 27]
```diff
@@ -0,0 +1,31 @@
...
+    $x = $y / 2
+    for ($i = 2; $i <= $x; $i++) {
+　　　　if ($y % $i === 0) {
+            return false; 
+        }
+    }
+    return true;
+}
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework3.php:17]: it should be
```
function Primenumber($y) {
    if ($y === 0 || $y === 1) {
        return false;
    }
    if ($y === 2) {
        return true;
    }
    $x = $y / 2;
    for ($i = 2; $i <= $x; $i++) {
        if ($y % $i === 0) {
            return false;
        }
    }
    return true;
}
```

File: php-homework/PHP-homework4.php [hunk 28]
```diff
@@ -0,0 +1,32 @@
...
+ }
+
+    $x = $y / 2 ;
+    for ($i = 2; $i <= $x; $i++) {
+        if ($y % $i === 0) {
+            return false; 
+        }
+    }
+    return true;
+}
```
Comment (pav-thanhtt) [inline php-homework/PHP-homework4.php:18]: save as above

**Comment by Misakiiii-333** [comment]
Good morning. I corrected mistakes about PHP-homework .

File: php-homework/PHP-homework4.php [hunk 29]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+function isPrimenumber($y) {
+    if ($y === 0 || $y === 1) {
+        return false; 
+ }
```
Comment (pa-vuhn) [inline php-homework/PHP-homework4.php:6]: The indent is wrong, please fix it.

File: php-homework/PHP-homework4.php [hunk 30]
```diff
@@ -0,0 +1,31 @@
...
+    if ($y === 2) {
+        return true; 
+ }
```
Comment (pa-vuhn) [inline php-homework/PHP-homework4.php:9]: This indent is wrong, please fix it.

File: php-homework/PHP-homework3.php [hunk 31]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+function isPrimenumber($y) {
+    if ($y === 0 || $y === 1) {
+        return false;
+ }
```
Comment (pa-vuhn) [inline php-homework/PHP-homework3.php:6]: This indent is wrong, please fix it.

File: php-homework/PHP-homework3.php [hunk 32]
```diff
@@ -0,0 +1,31 @@
...
+    if ($y === 2) {
+        return true;
+ }
```
Comment (pa-vuhn) [inline php-homework/PHP-homework3.php:9]: This indent is wrong, please fix it.

**Comment by Misakiiii-333** [comment]
Good morning. I corrected mistakes about PHP-homework .

**Pull request merged**